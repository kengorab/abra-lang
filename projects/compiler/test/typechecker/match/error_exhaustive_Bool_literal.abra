val b = true
val x = match b {
  true => 1
  false => 2
  _ => 3
}
