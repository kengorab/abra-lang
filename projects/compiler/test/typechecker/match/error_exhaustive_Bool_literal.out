Error at %FILE_NAME%:5:3
Unreachable match case
  |    _ => 3
       ^
The previous case(s) cover all possibilities for match target type 'Bool'
